app/models/manageiq/providers/azure/inventory/parser/cloud_manager.rb

```diff
@@ -463,10 +464,10 @@ def cloud_databases
       rg_ems_ref = collector.get_resource_group_ems_ref(database)
 
       persister.cloud_databases.build(
-        :ems_ref   => database.id,
-        :name      => "#{server.name}/#{database.name}",
-        :status    => server.properties&.user_visible_state,
-        :db_engine => "PostgreSQL #{server.properties&.version}",
+        :ems_ref        => database.id,
+        :name           => "#{server.name}/#{database.name}",
+        :status         => server.properties&.user_visible_state,
+        :db_engine      => "PostgreSQL #{server.properties&.version}",
         :resource_group => persister.resource_groups.lazy_find(rg_ems_ref)
       )
```

Comment:
Realized we don't have a lazy_find linking the cloud_database back to the server, https://github.com/ManageIQ/manageiq-schema/pull/653#issuecomment-1180774855

Reply:
That's a good idea, I'll add it in